RosBE-Windows/RosBE-Windows_RosBE.nsi

```diff
@@ -0,0 +1,526 @@
+    ;;
+    RMDir /r /REBOOTOK "$INSTDIR\i386"
+	RMDir /r /REBOOTOK "$INSTDIR\amd64"
+    RMDir /r /REBOOTOK "$INSTDIR\Bin"
+    RMDir /r /REBOOTOK "$INSTDIR\certs"
+    RMDir /r /REBOOTOK "$INSTDIR\samples"
+    RMDir /r /REBOOTOK "$INSTDIR\share"
+    StrCmp $ICONS_GROUP "" NO_SHORTCUTS
+    RMDir /r /REBOOTOK "$SMPROGRAMS\$ICONS_GROUP"
+    NO_SHORTCUTS:
+    Delete /REBOOTOK "$INSTDIR\Basedir.cmd"
+    Delete /REBOOTOK "$INSTDIR\Build.ps1"
+    Delete /REBOOTOK "$INSTDIR\Build-Shared.cmd"
+    Delete /REBOOTOK "$INSTDIR\ChangeLog.txt"
+    Delete /REBOOTOK "$INSTDIR\charch.cmd"
+    Delete /REBOOTOK "$INSTDIR\charch.ps1"
+    Delete /REBOOTOK "$INSTDIR\chdefdir.cmd"
+    Delete /REBOOTOK "$INSTDIR\chdefdir.ps1"
+    Delete /REBOOTOK "$INSTDIR\chdefgcc.cmd"
+    Delete /REBOOTOK "$INSTDIR\chdefgcc.ps1"
+    Delete /REBOOTOK "$INSTDIR\Clean.cmd"
+    Delete /REBOOTOK "$INSTDIR\Clean.ps1"
+    Delete /REBOOTOK "$INSTDIR\Help.cmd"
+    Delete /REBOOTOK "$INSTDIR\Help.ps1"
+    Delete /REBOOTOK "$INSTDIR\kdbg.cmd"
+    Delete /REBOOTOK "$INSTDIR\kdbg.ps1"
+    Delete /REBOOTOK "$INSTDIR\LICENSE.txt"
+    Delete /REBOOTOK "$INSTDIR\Make.cmd"
+    Delete /REBOOTOK "$INSTDIR\Makex.cmd"
+    Delete /REBOOTOK "$INSTDIR\options.cmd"
+    Delete /REBOOTOK "$INSTDIR\options.ps1"
+    Delete /REBOOTOK "$INSTDIR\playwav.ps1"
+    Delete /REBOOTOK "$INSTDIR\raddr2line.cmd"
+    Delete /REBOOTOK "$INSTDIR\raddr2lineNW.cmd"
+    Delete /REBOOTOK "$INSTDIR\README.pdf"
+    Delete /REBOOTOK "$INSTDIR\reladdr2line.ps1"
+    Delete /REBOOTOK "$INSTDIR\reladdr2lineNW.ps1"
+    Delete /REBOOTOK "$INSTDIR\Remake.cmd"
+    Delete /REBOOTOK "$INSTDIR\Remakex.cmd"
+    Delete /REBOOTOK "$INSTDIR\Remake.ps1"
+    Delete /REBOOTOK "$INSTDIR\Remakex.ps1"
+    Delete /REBOOTOK "$INSTDIR\Renv.cmd"
+    Delete /REBOOTOK "$INSTDIR\RosBE PS - PostInstall.reg"
+    Delete /REBOOTOK "$INSTDIR\RosBE.cmd"
+    Delete /REBOOTOK "$INSTDIR\rosbe.ico"
+    Delete /REBOOTOK "$INSTDIR\RosBE.ps1"
+    Delete /REBOOTOK "$INSTDIR\rosbe-gcc-env.cmd"
+    Delete /REBOOTOK "$INSTDIR\rosbe-gcc-env.ps1"
+    Delete /REBOOTOK "$INSTDIR\scut.cmd"
+    Delete /REBOOTOK "$INSTDIR\scut.ps1"
+    Delete /REBOOTOK "$INSTDIR\TimeDate.cmd"
+    Delete /REBOOTOK "$INSTDIR\uninstall.ico"
+    Delete /REBOOTOK "$INSTDIR\update.cmd"
+    Delete /REBOOTOK "$INSTDIR\update.ps1"
+    Delete /REBOOTOK "$INSTDIR\version.cmd"
+    Delete /REBOOTOK "$INSTDIR\version.ps1"
+    Delete /REBOOTOK "$INSTDIR\Uninstall.exe"
+    ;; Whoever dares to change this back into: RMDir /r /REBOOTOK "$INSTDIR" will be KILLED!!!
+    RMDir /REBOOTOK "$INSTDIR"
+
+    ;;
+    ;; Clean up the registry.
+    ;;
+    DeleteRegKey ${PRODUCT_UNINST_ROOT_KEY} "${PRODUCT_UNINST_KEY}"
+    DeleteRegKey HKCU "${PRODUCT_DIR_REGKEY}"
+    SetAutoClose true
+SectionEnd
+
+;; HACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACKHACK
```

Comment:
This might be a hack?

Reply:
This stuff is outdated and will be replaced soon